Changelog.md

```diff
@@ -15,6 +15,7 @@
  * Add conversions between ShortByteString and CString (#126)
  * Documentation fixes (#65, #118, #144, #150, #152, #172)
  * Resolve potential copyright issue with test data (#165)
+  * Don't perform unaligned writes when it isn't known to be safe (#133)
```

Comment:
The changelog entry should probably be added to the section for 0.10.10.1 above. This is the 0.10.10.0 section.

Reply:
Oops!

Reply:
Fixed

Reply:
Thanks!